src/v2/WidgetRouter.js

```diff
@@ -13,11 +13,13 @@
 import BaseLoginRouter from './BaseLoginRouter';
 import FormController from './controllers/FormController';
 import RegistrationFormController from './controllers/RegistrationFormController';
+import ForgotPasswordFormController from './controllers/ForgotPasswordFormController';
 
 module.exports = BaseLoginRouter.extend({
   routes: {
     '': 'defaultAuth',
     'signin/register': 'renderRegister',
```

Comment:
remind me why we are adding `signin` as prefix in routes ?

Reply:
It's for feature parity. 
If you asked for why v1 has `signin` prefix, I'm guess it's because all routes in v1 prefix with `signin`